OTKit/otkit-colors/token.yml

```diff
@@ -1,36 +1,201 @@
 props:
-  # Primary Action
-  # This color is only used on objects, backgrounds with large text (24px or larger, bold), and will be used as a color to guide users towards actionable content.
-  # =============================================
-  color-primary:
-    value: "#da3743"
-  color-primary-active:
-    value: "#b8222d"
+  ash-lightest:
+    value: "#f1f2f4"
 
-  # Primary Gray, used on all text sizes
-  # =============================================
-  color-gray-primary:
-    value: "#2d333f"
+  ash-lighter:
+    value: "#d8d9db"
+
+  ash-light:
+    value: "#91949a"
 
-  # Secondary Gray, used on smaller font sizes and for disabled primary buttons
-  # =============================================
-  color-gray-secondary:
+  ash:
     value: "#6f737b"
 
-  # Utility Gray, used for borders, dividers, containers
-  # =============================================
-  color-gray-utility:
-    value: "#d8d9db"
+  ash-dark:
+    value: "#2d333f"
 
-  # Lightest gray, used for backgrounds for the page and backgrounds for active/hover states
-  # =============================================
-  color-gray-active:
-    value: "#f1f2f4"
+  ash-darker:
+    value: "#141a26"
+
+  red-lightest:
+    value: "#fceeef"
+
+  red-lighter:
+    value: "#eea0a5"
+
+  red-light:
+    value: "#e15b64"
+
+  red:
+    value: "#da3743"
+
+  red-dark:
+    value: "#931b23"
+
+  red-darker:
+    value: "#450d10"
+
+  aqua-lightest:
+    value: "#eefcf9"
+
+  aqua-lighter:
+    value: "#3ddbb6"
+
+  aqua-light:
+    value: "#1fa888"
+
+  aqua:
+    value: "#18856b"
+
+  aqua-dark:
+    value: "#0c4134"
+
+  aqua-darker:
+    value: "#09342a"
+
+  blue-lightest:
+    value: "#eef1fc"
+
+  blue-lighter:
+    value: "#b1c1f1"
+
+  blue-light:
+    value: "#6c8ae4"
+
+  blue:
+    value: "#4a6fde"
+
+  blue-dark:
+    value: "#2146b5"
+
+  blue-darker:
+    value: "#0d1b45"
+
+  fuchia-lightest:
```

Comment:
`fuchia` ->
`fuchsia`

Reply:
LOL. Oh snap @mannionaco I think I pulled these by screenscraping the invision, you have to fix it there too ;)

Reply:
it's a nitpick, but one that will bother me if it stay in and gets put into use everywhere

Reply:
Fixed!